src/osx/textctrl_osx.cpp

```diff
@@ -843,4 +843,10 @@ wxTextProofOptions wxTextWidgetImpl::GetCheckingOptions() const
 void wxTextWidgetImpl::SetJustification()
 {
 }
+
+void wxTextCtrl::SetMaxLength(unsigned long length)
+{
+    if( HasFlag( wxTE_MULTILINE ) )
+        m_maxlen = length;
```

Comment:
What happens if the current length is greater? Should we truncate it or not? What does wxMSW do here?